src/odemis/gui/comp/overlay/view.py

```diff
@@ -616,10 +616,12 @@ def draw(self, ctx):
                 h_draw = True
                 v_draw = True
 
-            self.x_label = units.readable_str(val[0], self.cnvs.unit_x,
-                                              img.guess_sig_num_rng(self.cnvs.range_x, val[0]))
-            self.y_label = units.readable_str(val[1], self.cnvs.unit_y,
-                                              img.guess_sig_num_rng(self.cnvs.range_y, val[1]))
```

Comment:
there isn't any usage of the `img` anymore, since the only usage is removed here. Can we also remove the import from the file(line 31)?